src/radical/pilot/task_overlay/worker.py

```diff
@@ -254,7 +239,7 @@ def _eval(self, data):
         '''
 
         try:
-            out = eval(data['code'])
+            out = eval(data['code'])                                      # noqa
```

Comment:
If you add `# pylint:disable= eval-used` and `# pylint:enable= eval-used` after this line the linter will silence.

Reply:
Any idea why `noqa` does not work?   I am not sure if we should add two lines of linter control for one line of correct code.  Maybe disable that one for the file then?

Reply:
should be fixed...